resources/functionMap.php

```diff
@@ -8075,7 +8075,7 @@
 'php_logo_guid' => ['string'],
 'php_sapi_name' => ['__benevolent<non-empty-string|false>'],
 'php_strip_whitespace' => ['string', 'file_name'=>'string'],
-'php_uname' => ['string', 'mode='=>'string'],
+'php_uname' => ['"a"|"s"|"n"|"r"|"v"|"m"', 'mode='=>'string'],
```

Comment:
looks like your intension was to restrict the parameter. you have adjusted the return-type though